docs/source/developer_notes/graphics.rst

```diff
@@ -0,0 +1,71 @@
+Graphics
+========
+
+
+A ``Graphic`` is something that can be added to a ``PlotArea`` (described in detail in the layouts section). All the various
+fastplotlib graphics, such as ``ImageGraphic``, ``ScatterGraphic``, etc. inherit from the ``Graphic`` base class in
+``fastplotlib/graphics/_base.py``. It has a few properties that mostly wrap ``pygfx`` ``WorldObject`` properties and transforms.
+
+.. code-block:: rst
+
+    Graphic
+    │
+    ├─ ImageGraphic
+    │
+    ├─ TextGraphic
+    │
+    ├─ PositionsGraphic
+    │   │
+    │   ├─ LineGraphic
+    │   │
+    │   └─ ScatterGraphic
+    │
+    └─ GraphicCollection
+        │
+        └─ LineCollection
+            │
+            └─ LineStack
+
+..
+
+All graphics can be given a string name for the user's convenience. This allows graphics to be easily accessed from
+plots, ex: ``subplot["some_image"]``.
+
+All graphics contain a ``world_object`` property which is just the ``pygfx.WorldObject`` that this graphic uses. Fastplotlib
+keeps a *private* global dictionary of all ``WorldObject`` instances and users are only given a weakref proxy to this world object.
+This is due to garbage collection. This may be quite complicated for beginners, for more details see this PR: https://github.com/fastplotlib/fastplotlib/pull/160 .
+If you are curious or have more questions on garbage collection in ``fastplotlib`` you're welcome to post an issue :D.
+
+Graphic collections are groups of graphics. For now, we have a ``LineCollection`` which is a group of ``LineGraphic`` objects. We also have a ``LineStack`` which
```

Comment:
```suggestion
Graphic collections are collections of graphics. For now, we have a ``LineCollection`` which is a collection of ``LineGraphic`` objects. We also have a ``LineStack`` which
```

So that users don't get confused with `pygfx.Group` which is completely diff.